code/modules/food_and_drinks/food/customizables.dm

```diff
@@ -277,7 +277,7 @@
 
 /obj/item/reagent_containers/food/snacks/customizable/soup/Initialize()
 	. = ..()
-	eatverb = pick("slurp","sip","suck","inhale","drink")
```

Comment:
takes a succ

lmao

also
>inhales the soup

what the fuck

Reply:
Valid expression actually.